akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowPrependSpec.scala

```diff
@@ -5,8 +5,6 @@
 package akka.stream.scaladsl
 
 //#prepend
-import akka.stream.scaladsl.Source
-import akka.stream.scaladsl.Sink
```

Comment:
I guess there is a warning for this, but they are here to show the imports in the docs.
Please revert.